### PR TITLE
sapconf to set ALL specified values on OS irrespective of current val…

### PR DESCRIPTION
**Pull request by angelabriel**
…ue (not just increase) (fate#325548)

change variable names in sysconfig file for the avoidance of doubt (bsc#1070495)
remove unnecessary TMPFS_SIZE_MIN from sysconfig file (bsc#1070496)
correct spelling of NetWeaver

Please review. Hope, that SAP will be satisfied now. A test package will be sent immediately to Sören and Lee. The sapconf-test scripts for QA will be adjusted and extended next.

### CODE REVIEW

**Comment by diegoakechi** [comment]
@angelabriel  What do you think to include a comment on the /etc/sysconfig/sapconf file saying, that if the admin edit or remove values from this file, sapconf will no longer behave as designed to or monitor these edited parameters?

File: sysconfig/sapconf [hunk 1]
```diff
@@ -362,3 +340,4 @@ THP=never
 # UserTasksMax will be set to 'infinity'
 # A message will indicate if a reboot/restart is necessary.
 #
+#SAPCONF_END=""
```
Comment (diegoakechi) [inline sysconfig/sapconf:347]: If I am not mistaken, this line needs to be removed for upgrade purposes

File: lib/util.sh [hunk 2]
```diff
@@ -68,12 +68,36 @@ increase_val() {
...
     else
-	echo -n "$val2set"
+        echo -n "$val2set"
+    fi
+}
+
+# check, if a value is defined in the (sourced) sysconfig file
+# if not, log a warning and leave the current system value unchanged
+# otherwise set the value from sysconfig file as the new system value
+chk_and_set_conf_val() {
```
Comment (diegoakechi) [inline lib/util.sh:84]: I liked that we are still logging what decision was made by sapconf. It helps during debugging.

**Comment by angelabriel** [comment]
Good point. I will add
## This file includes parameters recommended by SAP
## Be in mind: If you edit or remove values from this file, sapconf will no 
## long behave as designed to or monitor these removed parameters
to the head of the sysconfig file

**Comment by diegoakechi** [comment]
@angelabriel Agreed.

**Review by diegoakechi — APPROVED** [review]
Now, It looks very good to me.

**Pull request merged**